src/libmongoc/doc/installing.rst

```diff
@@ -281,22 +290,7 @@ Open the MingGW shell with ``c:\msys64\ming64.exe`` (not the msys2_shell). Insta
   $ pacman --noconfirm -S mingw-w64-x86_64-extra-cmake-modules make tar
   $ pacman --noconfirm -S mingw64/mingw-w64-x86_64-cyrus-sasl
 
-Download and untar the latest tarball.
-
-.. code-block:: none
-
-  $ curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/x.y.z/mongo-c-driver-x.y.z.tar.gz
-  $ tar xzf mongo-c-driver-x.y.z.tar.gz
-  $ cd mongo-c-driver-x.y.z
-
-Create a build directory and build and install with cmake:
-
-.. code-block:: none
-
-  $ mkdir cmake-build
-  $ cd cmake-build
-  $ CC=/mingw64/bin/gcc.exe /mingw64/bin/cmake -G "MSYS Makefiles" -DCMAKE_INSTALL_PREFIX="C:/mongo-c-driver" -DCMAKE_C_FLAGS="-D__USE_MINGW_ANSI_STDIO=1" ..
-  $ make install
+Follow the build instructions above.
```

Comment:
Here, it seems like we may not be able to redirect users. The line:
```
CC=/mingw64/bin/gcc.exe /mingw64/bin/cmake -G "MSYS Makefiles" -DCMAKE_INSTALL_PREFIX="C:/mongo-c-driver" -DCMAKE_C_FLAGS="-D__USE_MINGW_ANSI_STDIO=1" ..
```
The generator and `-D__USE_MINGW_ANSI_STDIO=1` is specific to MinGW.

In that case, I'm fine with having the duplication of instructions here, and just replacing `make install` with `cmake --build . --target install` for consistency.

Reply:
I think my restructuring takes care of this.